lib/streamlit/server/server_util.py

```diff
@@ -104,6 +104,20 @@ def is_url_from_allowed_origins(url: str) -> bool:
     return False
 
 
+def _get_max_message_size_bytes() -> int:
+    """Returns the max websocket message size in bytes.
+
+    This will lazyload the value from the config and store it in the global symbol table.
+    """
+
+    if "server.maxMessageSize" not in globals():
+        # Load and convert the config value to bytes.
+        globals()["server.maxMessageSize"] = config.get_option(
```

Comment:
nit: I believe it's more common in the codebase to use the `global` keyword to mutate a variable defined in the module global scope to do this type of caching (although I'm fairly certain the two approaches are equivalent, so there's an argument to be made for keeping this as-is if we want to keep the name the same)

See 

* `local_sources_watcher.py` L89
* `elements/utils.py` L65
* `config.py` L1061

for some examples

Reply:
Thanks for the examples! My intention here was to keep the lazyloading function self-contained. I think the `global` keyword requires the variable to be defined/initialized with `None` on the module level (outside of the function). However, since there are already a few other lazyloading examples in the codebase with `global` keyword, I will adapt that as well here :)